Polly/src/scenes/CreatePoll/CreatePollScreen.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { ThemeProvider, Button } from 'react-native-elements'
+import { View, StyleSheet } from 'react-native'
+import { createPoll } from '../../actions/polls'
+import { theme, elevationShadowStyle } from './index'
+
+class CreatePollScreen extends React.Component {
```

Comment:
I would create a CreatePollContainer class in a separate final, who's only job is to render CreatePollScreen but also connect to the redux-store with mapDispatchToProps and then pass the createPoll function in as prop to CreatePollScreen